src/qt/clientmodel.h

```diff
@@ -94,6 +96,8 @@ class ClientModel : public QObject
 
     bool getTorInfo(std::string& ip_port) const;
 
+    static void mapPort(bool use_upnp) { StartMapPort(use_upnp); }
```

Comment:
Better to move the implementation to the cpp file and make it a class method. The `mapport.h` include in the `clientmodel` header file makes that all of the `mapport.h` functions are visible and accesible by every GUI object that has access to `clientModel`, going against the MVC structure.
Plus, be a class method instead of an static function will let us implement different models in the future as well.

Reply:
i've moved the implementation to the cpp file and also moved the `mapport.h` include there as well, but have kept it as a static function to appease clang-tidy.

if, in the future this needs to be expanded beyond what a static function will allow, it can be changed then.